.editorconfig

```diff
@@ -0,0 +1,8 @@
+root = true
+
+[*]
+indent_style = space
+indent_size = 2
+trim_trailing_whitespace = true
+end_of_line = lf
```

Comment:
Yes, this repo absolutely needs to be running CI testing, I'm a bit amazed it hasn't already been set up for that.